### DISCUSSION TITLE
How to click input type=radio

### DISCUSSION THREAD

**Discussion by Pafnodeveloper** [webdriverio/webdriverio#7582] [Q&A] [answered]
Hi, everyone. I am new in WDIO so I came across a problem. There are two input type=radio and I need to switch from one that 'checked' to other one.
![two radio](https://user-images.githubusercontent.com/51054260/138553329-33222621-716d-47cc-b0ab-26a9fe84a46b.png)
As you can see on the screenshot when I move my cursor on the needed element there is no any indication of its presence on the user interface. Subsequently when I try to click it I get the next error "Node is either not clickable or not an HTMLElement" but my previous line on my code is console.log(purgeCacheLabel.getAttribute("value")) and the output is "COMMAND getElementAttribute("ELEMENT-10", "value") RESULT all".  So it actually exists and was found. I'll appreciate any help

**Comment by christian-bromann** [comment, marked as answer]
So you are saying that:

```js
const purgeCacheLabel = $('some selector')
console.log(purgeCacheLabel.getAttribute("value")) // outputs "all"

// but then
purgeCacheLabel.click() // throws "Node is either not clickable or not an HTMLElement"
```
?

**Reply by Pafnodeveloper** [reply]
Yep, that's exactly what i am saying. And one important thing you missed. The element get purgeAllCacheLabel() {return $('#all')} isn't highlited on the interface when I click or choose it on DOM

**Reply by christian-bromann** [reply]
Can you provide a reproducible example? Also, can you try to click on the label instead?

**Reply by Pafnodeveloper** [reply]
It worked for label, very interesting! But, why? Is it clickable tag? And what does this #shadow-root (user-agent) inside needed input mean?

**Reply by christian-bromann** [reply]
> But, why?

Not sure if you are using a web component library that maybe interferes with how the driver simulates the click.
